docs/no-sizzle.md

```diff
@@ -0,0 +1,114 @@
+# no-sizzle
+
+Disallows selector extensions provided by Sizzle. Use the `allowPositional` option to allow positional selectors. Use the `allowOther` option to allow all other selectors. These options are used in the `deprecated-3.4` ruleset as only positional selectors were deprecated in that version.
+
+## Rule details
+
+✗ The following patterns are considered errors:
+✗ With `[{"allowPositional":true}]` options:
+```js
+$( ':animated' );
+```
+```js
```

Comment:
why is there 2 code blocks here ?

Reply:
The second code block is unlabelled as there are no options. I have a follow-up to change the order so it reads more sensibly.